backend/gene_info/api/v1.py

```diff
@@ -10,15 +15,18 @@ def gene_info(geneID: string):
     try:
         uid = provider.fetch_gene_uid(geneID)
     except NCBIAPIException:
-        return make_response(jsonify("Failed search of NCBI database, API key issue"), 404)
+        logging.error("Failed search of NCBI database, API key issue")
+        raise ForbiddenHTTPException
```

Comment:
remove the log above and put the message as the first parameter to the exception. Do this for all of your exceptions.
```suggestion
        raise ForbiddenHTTPException("Failed search of NCBI database, API key issue")
```